core/src/main/resources/error/error-classes.json

```diff
@@ -63,5 +63,86 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "INVALID_INSERT_INTO_CONTEXT" : {
```

Comment:
These need to be in alphabetical order to pass the style tests.

Reply:
I got it.